org.metaborg.spt.cmd/src/main/java/org/metaborg/spt/cmd/Module.java

```diff
@@ -25,4 +37,29 @@
     @Override protected void bindEditor() {
         bind(IEditorRegistry.class).to(NullEditorRegistry.class).in(Singleton.class);
     }
+
+    @Override
+    protected void bindTestReporter() {
+        Class<? extends ITestReporterService> reporterClass = getClassByNameOrDefault(this.customReporterClassName, LoggingTestReporterService.class);
+        // Bind the (custom or default) reporter.
+        bind(ITestReporterService.class).to(reporterClass).in(Singleton.class);
+    }
+
+    private <T> Class<? extends T> getClassByNameOrDefault(@Nullable String className, Class<? extends T> defaultClass) {
+        if (className == null)
+            return defaultClass;
+        try {
+            return getClassByName(className);
+        } catch (ClassNotFoundException | ClassCastException e) {
+            // Fallback to default.
+            throw new RuntimeException(e);
+            // TODO: Return default instead of throwing an error:
+            // TODO: Should log when given class could not be loaded!
+//            return defaultClass;
```

Comment:
I'm not sure. What would be the best course of action when an error occurs in an injection module's `configure()` method? Currently I throw a RuntimeException to avoid the check exceptions, and I assume I can't log since the logger hasn't been injected yet.